### DISCUSSION TITLE
unsupport_chain

### DISCUSSION THREAD

**Discussion by wreqwewqe** [wevm/wagmi#3208] [Q&A]
i want to know how to change to scroll, now is doesn't support scroll

**Comment by POKENA7** [comment]
You can import scroll info from `viem/chains` and set it to `configuraChains`.
(requires [viem v1.16.4](https://github.com/wagmi-dev/viem/releases/tag/viem%401.16.4) or higher)
```
import { scroll } from "viem/chains";
import { publicProvider } from "wagmi/providers/public";

export const { chains, publicClient } = configureChains(
  [scroll],
  [publicProvider()]
);
```
